test/test_update.py

```diff
@@ -11,6 +11,14 @@
 from test.helper import FakeYDL, report_warning
 from yt_dlp.update import Updater, UpdateInfo
 
+
+# XXX: Keep in sync with yt_dlp.update.UPDATE_SOURCES
+TEST_UPDATE_SOURCES = {
+    'stable': 'yt-dlp/yt-dlp',
+    'nightly': 'yt-dlp/yt-dlp-nightly-builds',
+    'master': 'yt-dlp/yt-dlp-master-builds',
+}
```

Comment:
Why not just import from there?

Reply:
So that we can test more extensively and forks that modify the actual `UPDATE_SOURCES` won't have failing CI (e.g. updater tests fail on my actions-test branches currently, unless I were to manually modify each and every failing test case)